### PR TITLE
chore(docs): add custom marker documentation

### PR DESCRIPTION
**Pull request by awitherow**
add documentation about how to add custom markers, derived from discussion in https://github.com/xkjyeah/vue-google-maps/issues/38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,57 @@ Or use the power of Vue.js within a google map like this:
...
+	        panControlOptions: false,
+	        gestureHandling: 'greedy'
+        },
+    }
+})
+export default settings;
+
+loaded.then(() => {
+    /* google.maps.Size is now available! */
+    settings.defaultIconSettings = /* Do something with google.maps.Size */
```
Comment (awitherow) [inline README.md]: @xkjyeah could you elaborate here what other things are available here via the google.maps object? or at least point to documentation somehow.

Comment (awitherow) [inline README.md]: @xkjyeah 👍

Comment (xkjyeah) [reply README.md]: er... to be honest the Google Maps API surprises me with every new version.

The link is https://developers.google.com/maps/documentation/javascript/3.exp/reference

Comment (awitherow) [reply README.md]: @xkjyeah I can add a link to that so we everyone has an idea of what the options are.

**Comment by awitherow** [comment]
@xkjyeah I've updated the documentation where you requested, adding simply a link for the user to go looking for what they can access themselves. Looks good?

File: README.md [hunk 2]
```diff
@@ -89,6 +89,60 @@ Or use the power of Vue.js within a google map like this:
...
+    // the google.maps object.
+    settings.defaultIconSettings = {};
+})
+```
+
+(1) In the [Google Maps Javascript Docs](https://developers.google.com/maps/documentation/javascript/3.exp/reference), you can find all of the options available in the google.maps object.
+
+**yourMap.vue**
+
+```javascript
```
Comment (xkjyeah) [inline README.md:128]: Either \`\`\`vue (if Github supports it) or \`\`\`html

File: README.md [hunk 3]
```diff
@@ -89,6 +89,60 @@ Or use the power of Vue.js within a google map like this:
...
+          gestureHandling: 'greedy'
+        },
+    }
+})
+export default settings;
+
+loaded.then(() => {
+    // (1) see note below for information about what is available from
+    // the google.maps object.
+    settings.defaultIconSettings = {};
```
Comment (xkjyeah) [inline README.md:120]: I think it is worth showing how `google.maps` can be used here, e.g. for `google.maps.Size`.

File: README.md [hunk 4]
```diff
@@ -89,6 +89,60 @@ Or use the power of Vue.js within a google map like this:
 </script>
 ```
 
+## Custom markers
+
+To implement custom marker, best practice is to create a map settings file, which you can import into the page(s) where you use the gmap-map and gmap-marker tags. This is usually done in a project configuration folder.
```
Comment (xkjyeah) [inline README.md:94]: It is worth explaining the problem, that is, why:

```html
<gmap-marker :icon="{... size: new google.maps.Size(10,10)}"></gmap-marker>
```

does not work

**Review by xkjyeah — CHANGES_REQUESTED** [review]

**Comment by diegoazh** [comment]
Please resolve the comments and conflicts in order to be merged @awitherow. Thank you.

**Comment by AllanOcelot** [comment]
@diegoazh  - Last push was February 2018, I would vote to close this PR.
Given the two years, there's also a good chance this would need updating for changes in GM.

**Comment by diegoazh** [comment]
@AllanOcelot you are right. I will test this with actual changes and if its not working I will close this PR.
I think maybe this work yet 😐

**Comment by diegoazh** [comment]
I think that this PR should be closed for inactivity.

**Comment by diegoazh** [comment]
@AllanOcelot can you review this work and update it on a new PR? I think the author of this PR is no longer interested on this library.

**Pull request closed without merging**